scripts/npc/npc_abilities_custom.txt

```diff
@@ -8682,15 +8682,13 @@
 					"MODIFIER_PROPERTY_BASEDAMAGEOUTGOING_PERCENTAGE"	"%percentage_bonus_damage"
 					"MODIFIER_PROPERTY_PHYSICAL_ARMOR_BONUS"		"%bonus_armor"
 				}
+			}
```

Comment:
There were several missing closing brackets in the file which we somehow were not seeing the effects of due to the -- comments already corrupting K/V data